core/modules/ccontrol/MergingRequester.cc

```diff
@@ -227,11 +227,10 @@ bool MergingRequester::_merge() {
 }
 
 void MergingRequester::_setError(int code, std::string const& msg) {
-    LOGF_INFO("setError");
-    LOGF_INFO("setError %1% %2%" % code % msg);
+    LOGF_INFO("setError: code: %1%, message: %2%" % code % msg);
     std::lock_guard<std::mutex> lock(_errorMutex);
-    _error.code = code;
-    _error.msg = msg;
+    _error.setCode(code);
+    _error.setMsg(msg);
```

Comment:
I agree this clearer, but using `_error = Error(code, msg);` will create one additional error object (_error is created with default constructor while constructing parent object) and call copy constructor. Is it as performant than calling setters?